assembl/models/path_utils.py

```diff
@@ -456,16 +492,24 @@ def copy_result(self, idea_id, parent_result, child_result):
         self.viewed_counts[idea_id] = parent_result.viewed_count
 
     def get_counts_for_query(self, q):
+        # HACKITY HACK
```

Comment:
This is outside the scope of the pull request. I'd be happy to document the whole thing, but what's Bluenove budget for that?